.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thanks for submitting a pull request! Before pressing that tempting green submit
+button, please make sure you check off all the following boxes:
+
+- [ ] All existing unit tests pass in all supported Python versions (2.6, 2.7,
```

Comment:
Maybe just say 'All existing unit tests pass' to remind people to test before submitting, and let travis handle the versions?